UnityProject/Assets/Scripts/Construction/Managers/Deconstruction.cs

```diff
@@ -63,15 +63,8 @@ private void DoWallDeconstruction(Vector3Int cellPos, TileChangeManager tcm, Vec
 		tcm.RemoveTile(cellPos, LayerType.Walls);
 		SoundManager.PlayNetworkedAtPos("Deconstruct", worldPos, 1f);
 
-		//Spawn 2 metal sheets:
-		int spawnMetalsAmt = 0;
-		while (spawnMetalsAmt < 2)
-		{
-			spawnMetalsAmt++;
-			PoolManager.PoolNetworkInstantiate(metalPrefab, worldPos, tcm.transform);
-		}
-
-		//TODO spawn wall girder!
+		PoolManager.PoolNetworkInstantiate(metalPrefab, worldPos, tcm.transform);
+		PoolManager.PoolNetworkInstantiate(wallGirderPrefab, worldPos, tcm.transform);
```

Comment:
For some reason wallGirderPrefab is missing:

![image](https://user-images.githubusercontent.com/20813925/56716666-3ee13e80-677e-11e9-9696-af98b360a969.png)